lib/internal/Magento/Framework/Session/SessionManager.php

```diff
@@ -504,11 +501,7 @@ public function regenerateId()
             return $this;
         }
 
-        if ($this->isSessionExists()) {
-            session_regenerate_id(true);
-        } else {
-            session_start();
-        }
+        $this->isSessionExists() ? session_regenerate_id(true) : session_start();
```

Comment:
Ternary operator is not a good idea when there is no return value.

By the way, in terms of code calisthenics ternary operator is still else :)